Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM openjdk:7-jre
 
-# install `make`
-RUN apt-get -y update && apt-get install -y -q build-essential
+# install make and mvn
+RUN apt-get -y update && apt-get install -y -q build-essential maven
```

Comment:
the jar-downloading runs here in the Dockerfile, and the compiling of the go binary happens outside the container (it used to run inside but was moved out).